test/stat_tracker_test.rb

```diff
@@ -227,4 +227,16 @@ def test_it_knows_wins_by_season
     expected = {3=>0, 5=>1, 10=>0, 26=>1, 22=>0}
     assert_equal expected, @stat_tracker.wins_in_season(20152016)
   end
+
```

Comment:
We'll probably need to change a lot of these test results to be sure they work with newer fixture files.